include/crpropa/magneticField/turbulentField/PlaneWaveTurbulence.h

```diff
@@ -0,0 +1,110 @@
+#ifndef CRPROPA_PLANEWAVETURBULENCE_H
+#define CRPROPA_PLANEWAVETURBULENCE_H
+
+#include "crpropa/Grid.h"
+#include "crpropa/magneticField/turbulentField/TurbulentField.h"
+#include <vector>
+
+namespace crpropa {
+/**
+ * \addtogroup MagneticFields
+ * @{
+ */
+
+/**
+ @class PlaneWaveTurbulence
+ @brief Interpolation-free turbulent magnetic field based on the GJ99 and TD13
+papers
+
+ ## Overview
+ This class provides a turbulent magnetic field that is generated as described
+by [(Giacalone and Jokipii, 1999)][gj99] and [(Tautz and Dosch, 2013)][td13].
+Instead of using an inverse Fourier transform to generate the field on a grid --
+which then needs to be interpolated to obtain in-between values -- this method
+only generates the wave modes making up the turbulent magnetic field ahead of
+time. At run time, when the field's value at a particular position is required,
+these plane waves are then evaluated analytically at that position. This
+guarantees that the resulting field is completely free of divergence, reproduces
+the mean field strength accurately, and does not suffer from other
+interpolation-induced problems. The disadvantage is that the number of wave
+modes is drastically smaller when compared with initTurbulence, which might have
+physical ramifications on the particles propagating through the field.
+Additionally, the implementation is somewhat slower.
+
+ ## Using the SIMD optimization
+ In order to mitigate some of the performance impact that is inherent in this
+method of field generation, an optimized version utilizing data-level
+parallelism through SIMD instructions is provided. More specifically, this
+implementation uses the AVX extension to the x86 instruction set. In order to
+use this optimized version, three conditions need to be met:
+
+1. The `USE_SIMD` option needs to be explicitly enabled in CMake. Currently,
+this sets GCC flags that tell the compiler to allow SIMD instructions.
```

Comment:
Make this sentence passive, too.
The `FAST_WAVES` option needs to be enabled...

Reply:
I think I did this to avoid doubling the "needs to be enabled" construction, but I can understand that switching to active voice might seem weird or clunky, too.

Reply:
Changed.